lib/Switch.js

```diff
@@ -24,16 +24,16 @@ export class Switch extends Component {
     circleSize: PropTypes.number,
     circleBorderActiveColor: PropTypes.string,
     circleBorderInactiveColor: PropTypes.string,
-    activeTextStyle: PropTypes.object,
-    inactiveTextStyle: PropTypes.object,
-    containerStyle: PropTypes.object,
+    activeTextStyle: Text.propTypes.style,
```

Comment:
This will not work on the newest React-native Version. Rollback this changes, please.